spec/models/document_spec.rb

```diff
@@ -15,6 +15,19 @@
 
       expect(document.public_timestamp).to be_nil
     end
+
+    it 'does not break external links' do
```

Comment:
It might be helpful to have more context here. Why might external links break?

Reply:
Your guess is as good as mine!  😂